src/main/webapp/resources/js/components/Tables/fitlers/TextFilter.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
```

Comment:
File comment

Reply:
Updated in 3ce9d15384edebcbbc46221a282033890d0ec604